modules/library/referencing/src/main/java/org/geotools/referencing/operation/matrix/Matrix2.java

```diff
@@ -31,7 +32,6 @@
  * @source $URL$
  * @version $Id$
  * @author Martin Desruisseaux (IRD)
- * @deprecated Use GeneralMatrix
  */
```

Comment:
> Matrix2 has been marked as deprecated, yet Eclipse still finds 22 references to it. If you deprecate a class it would be good to clean the source code of its usage, no?
> Same goes for Matrix3 and Matrix4... wondering if the deprecation notes should just be removed?
> Otherwise we are just getting into the Filter business again, years and years of deprecated classes without a (funded) plan to remove them.

Matrix2 is still holding its own weight (as an optimized implementation) and I suspect we will do the same for Matrix3. Marking them as non deprecated for now.

Interestingly I found the "fast" implementation we could be using behind this class - FixedMatrix2x2_64F. Looks like there is a host of auto generated implementations for fixed matrixes that we could be using instead of DenseMatrix64F.